src/frontend/src/views/Upload/components/Metadata/components/StaticTable/components/Row/style.ts

```diff
@@ -0,0 +1,80 @@
+import styled from "@emotion/styled";
+import {
+  TableCell,
+  TableRow,
+  TableRowProps as MuiTableRowProps,
+} from "@mui/material";
+import { CommonThemeProps, fontHeaderS, getColors, getSpaces } from "czifui";
+import { rightMarginXxs } from "src/common/styles/iconStyle";
+
+export const Id = styled.div`
+  ${fontHeaderS}
+
+  min-width: 300px;
+
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+
+  ${(props: CommonThemeProps) => {
+    const spaces = getSpaces(props);
+
+    return `
+      padding: ${spaces?.m}px ${spaces?.m}px ${spaces?.m}px ${spaces?.s}px;
+    `;
+  }}
+`;
+
+interface TableRowProps extends MuiTableRowProps, CommonThemeProps {
+  component: "div";
+}
+
+export const StyledTableRow = styled(TableRow)`
+  &:nth-of-type(even) {
+    ${(props: TableRowProps) => {
+      const colors = getColors(props);
+
+      return `
+        background-color: ${colors?.gray[100]};
+      `;
+    }}
+  }
+`;
+
+export const StyledTableCell = styled(TableCell)`
+  max-width: 300px;
+  word-break: break-word;
+  min-width: 150px;
+  ${(props: CommonThemeProps) => {
+    const spaces = getSpaces(props);
+
+    return `
+      padding: ${spaces?.m}px;
+    `;
+  }}
+`;
+
+export const IsPrivateTableCell = styled(TableCell)`
```

Comment:
nit/opinion: I usually associate variables that start with "Is" with being booleans.  Can this be renamed to `PrivateTableCell`?